Test/Unit/Constants.php

```diff
@@ -169,7 +169,7 @@ private function addBillingAddress($cardId)
                 "firstname" => "asdf",
                 "lastname" => "MyParcel- asdfsd",
                 "saveInAddressBook" => null,
-                "email" => "reindert-test@myparcel.nl",
+                "email" => "reindert-test@sendmyparcel.be",
```

Comment:
uhhh, waar is dit voor? Mag mijn e-mail hier weg?

Reply:
Geen idee waarom jij dat in 2017 had gemaakt, maar je naam is aangepast